source/assets/js/components/fef-swipeable-area.js

```diff
@@ -409,104 +341,101 @@ export class FefSwipeableArea {
      * right-most) item and use it as the "target" item to center.
      */
     pageForward() {
-        let containerWidth = this.$innerContainer.innerWidth(),
-            visibleAreaRightEdge = this.$innerContainer.scrollLeft() + containerWidth,
-            partiallyVisibleItemIndex = this.itemPositions.findIndex(pos => pos.right > visibleAreaRightEdge),
-            targetItemIndex = this.getTargetItemIndex(partiallyVisibleItemIndex, 'forward');
+        this.track('click-right');
 
-        // Make sure index is not out of bounds
-        targetItemIndex = Math.min(targetItemIndex, this.itemPositions.length - 1);
+        if (SUPPORTS_SNAP_POINTS) {
```

Comment:
da safari weder auf desktop noch mobile nativ smooth-scrolling beherrscht, könnte man diesen browser auch in den "else"-case gehen lassen. vielleicht mit `CSS.supports('scroll-behaviour', 'smooth');` checken.

Reply:
Hab mich jetzt fast zu Tode gegoogelt warum da alle Browser `false` ausgeben. Aber es liegt an deinem britischen Englisch ;) `behaviour` vs `behavior`
Aber es funktioniert!